orc8r/gateway/python/magma/magmad/config_manager.py

```diff
@@ -145,4 +146,29 @@ def did_mconfig_change(serv_name):
             if srv in mconfig.configs_by_key:
                 configs_by_key[srv] = mconfig.configs_by_key.get(srv)
 
+        # version should be in 1.1.1 format with only positive numbers allowed
+        VERSION_REGEX = re.compile(r"[0-9]+\.(?P<minor_version>[0-9]+)\.[0-9]+")
+
+        agw_version = self._magmad_service.version
+        unpacked_mconfig = mconfig.configs_by_key.get(MAGMAD)
+        if unpacked_mconfig.Is(mconfigs_pb2.MagmaD.DESCRIPTOR):
```

Comment:
would we ever hit the corresponding else case? might be worth a log if it's unexpected

Reply:
it could since the config will be streamed by Orc8r, if it decides to stream down a structure other than MagmaD, maybe we can hit the else case, unless we have some protection built-in to the code when we process updates on magmad?